content/docs/releases/v1.2.0.md

```diff
@@ -0,0 +1,63 @@
+
+## Changes since v1.1.0
+
+### Policy Based Routing
+
+NSE's may now request [Policy Based Routing](https://en.wikipedia.org/wiki/Policy-based_routing) for NSCs that connect to them.  This is primarily envisioned to be used to request Source Based routing for the Src IP issued to the NSC by the NSE.  This enables routing the return traffic for incoming TCP connections over the vWire back out the vWire without having to necessarily know the full set of prefixes from which such traffic might originate.   Various restrictions apply to prevent NSE's requesting Policy Based Routing that conflicts with K8s Cluster Networking or with other Network Services.  See [api#119](https://github.com/networkservicemesh/api/issues/119) for more details.
+
+[Try NSM Policy Based Routing](https://github.com/networkservicemesh/deployments-k8s/tree/v1.2.0/examples/features/policy-based-routing)
+
+### Advanced Healing Cases
+
+NSM v1.2.0 is now testing with [advanced healing examples](https://github.com/networkservicemesh/deployments-k8s/tree/v1.2.0/examples/heal) including simultanoues failure of two components.
+
+### Open Telemetry Integration
+
+NSM v1.2.0 now provides OpenTelemetry Integration including both tracing and Prometheus Metrics.
+
+[Try NSM OpenTelemetry Integration](https://github.com/networkservicemesh/deployments-k8s/tree/v1.2.0/examples/observability)
+
+### Interdomain Wireguard Examples
+
+Interdomain examples using Wireguard for IP transport were added.
+
+[Try NSM Interdomain Wireguard Examples](https://github.com/networkservicemesh/deployments-k8s/tree/v1.2.0/examples/interdomain)
+
+### GKE Compatibility Bug Fixed
+As noted in [NSM v.1.1.0's Release Notes](/docs/releases/v1.1.0/#gke-compatibility), GKE had manifested an odd [Netlink IP Neighbor bug with IPv6](https://github.com/networkservicemesh/integration-k8s-gke/issues/215) in its latest node-version.  This issue has since been fixed, and NSM v1.2.0 works fine with up-to-date GKE node-versions.
+
+### Exclude prefixes already in use in a Pod by other Network Services
+When a Pod requests a Network Service, exclude prefixes now includes not only K8s Cluster prefixes, but also prefixes already in use from previously established connections to other Network Services by that Pod.
+
+[Try NSM Exclude Prefix example](https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/features/exclude-prefixes-client)
+
+### Select Forwarder Example
+
+In v1.1.0 NSM introduced the ability to select forwarders based on 'labels'.  v1.2.0 provides an example of that functionality.
+[Try NSM Forwarder Selection](https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/features/select-forwarder)
+
+### Stability
+- Healing - improved Registry healing
```

Comment:
```suggestion
- Registry events serialization rework. It allowed to NSM work correctly in case of death 2 or more components and also huge reduces registry memory allocation. 
```

Reply:
This comment is minor and can be ignored.